virtcontainers/factory/template/template.go

```diff
@@ -93,7 +94,7 @@ func (t *template) prepareTemplateFiles() error {
 		return err
 	}
 	flags := uintptr(syscall.MS_NOSUID | syscall.MS_NODEV)
-	opts := fmt.Sprintf("size=%dM", t.config.HypervisorConfig.MemorySize+8)
+	opts := fmt.Sprintf("size=%dM", t.config.HypervisorConfig.MemorySize+templateDeviceStateSize)
```

Comment:
This change is specifically for arm64 so although it won't stop the feature working for other architectures, it clearly will have an impact. In summary, why not create a `virtcontainers/factory/template/template_arm64.go` with the modified value specifically for arm64?